setup.py

```diff
@@ -65,9 +65,9 @@
     version=__version__,
     entry_points={
         "console_scripts": [
-            "mbedtools=mbed_tools.cli.cli:cli",
-            "mbed-tools=mbed_tools.cli.cli:cli",
-            "mbed_tools=mbed_tools.cli.cli:cli",
+            "mbedtools=mbed_tools.cli.main:cli",
```

Comment:
Ah, great. You give a better name here.

Reply:
Do you think we should do the "anything `mbed-` becomes a subcommand of mbedtools" thing? For example, when we want to allow others to plug in new commands. Do you think the commit "Remove sub package CLIs" makes this easier or harder to do in the future? I suspect we'd want each sub command to expose their own cli and be installed as e.g. "mbed-configure" for "mbedtools configure".

Reply:
Moving our internal commands shouldn't make it any harder to do that. If we use `https://pypi.org/project/click-plugins/` we only need to add the plugin commands to the main group. We should do it, but not as part of this merge.

Reply:
I'm not sure we want the packages that rely on core APIs (which are `targets`, `lib` and `project`) to be plugin modules (`configure` and `list_devices` both rely on these). If we want `configure` et al to actually be plugins, we shouldn't merge them into the same repo, and we should go back to the old structure of independent repos for every subpackage. Standalone packages (possibly `pyocd` and any others) could be converted to plugins and would be installed separately. Thoughts?

Reply:
I agree it makes sense to have some core commands that aren't implemented as plugins. Let's consider which commands make most sense as plugins after we 1) reduce the coupling between our submodules further, and 2) have the plugin mechanism determined.